src/preferences/colorpalettesettings.h

```diff
@@ -9,13 +9,17 @@ class ColorPaletteSettings {
             : m_pConfig(pConfig) {
```

Comment:
Add documentation above:
"Saves ColorPalettes to and loads ColorPalettes from the mixxx.cfg file"